src/parser/bison_parser.y

```diff
@@ -438,20 +441,25 @@ execute_statement : EXECUTE IDENTIFIER {
 /******************************
  * Import Statement
  * IMPORT FROM TBL FILE 'test/students.tbl' INTO students
- * COPY students FROM 'test/students.tbl' [WITH FORMAT TBL]
+ * COPY students FROM 'test/students.tbl' WITH (FORMAT TBL, ENCODING 'Dictionary')
```

Comment:
Do I get that right that the `WITH ...` part is now required?

Reply:
In the end, this then becomes sth like: 
```SQL
COPY students FROM 'test/students.tbl' [ [WITH] ( [FORMAT TBL] [, ENCODING 'Dictionary'] ) ]
```
and is more the grammer than an example (not to talk about the different options here and that the options' order can be changed). I was not sure what to do there tbh. We could as well just ad a third example with format and encoding. We will also deliberately use the encoding option totally different from what it was meant to be, but that's not to matter here.

Have you seen the change to the  `AlterAction` constructor? I noticed we do not care about shadowing in the parser at all. Looking back at why we changed it in Hyrise (hyrise/hyrise#1877) we should not have issues in the parser, but what's your opinion on applying Hyrise's style here as well?

Reply:
Any style improvements are welcome, sure. Can this be fixed automatically? Otherwise I am not sure if it is worth the effort.

Reply:
And your call on the example. Maybe a third sounds good, before we have too much grammar-stuff in the examples.

Reply:
It does not seem to be automatic, at least it does not seem to happened automatically in hyrise/hyrise#2040. With `-Weverything`, we have 35 occurrences of that. It's a rather rase case when this causes problems and I am a bit reluctant when it comes to adding errors looking at how many forks there are. So maybe leaving it as is is fine.

While playing arounf with it, I noticed that actually `-Wall -Werror` wad turned off for two years due to a typo in the flag name (and yes, again 100% my bad :/). I'll sneak in a fix.

Also added the extra example.